[Monitor - Opentelemetry Exporter] Execute Samples failing in nightly runs

Monitor - Opentelemetry Exporter nightly test runs are failing with:

> Error message:
>Found configuration in D:\a\_work\1\s\rush.json

>Rush Multi-Project Build Tool 5.63.0 - Node.js 12.22.12 (LTS)
Error: The command "execute:samples" is not defined in the package.json file for this project.
Available commands for this project are: "build", "build:browser", "build:node", "build:samples", "build:test", "check-format", "clean", "extract-api", "format", "functional-test", "generate:client", "integration-test", "integration-test:browser", "integration-test:node", "lint", "lint:fix", "pack", "prepare", "report", "test", "test-opentelemetry-versions", "test:browser", "test:node", "unit-test", "unit-test:browser", "unit-test:node", "unit-test:node:debug", "unit-test:node:no-timeout"
Use "rushx --help" for more information.
##[error]Cmd.exe exited with code '1'.

For more details check here:

- https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1646128&view=results

--- a/sdk/monitor/monitor-opentelemetry-exporter/samples-dev/httpSample.ts
+++ b/sdk/monitor/monitor-opentelemetry-exporter/samples-dev/httpSample.ts
@@ -77,6 +77,7 @@ function handleRequest(request: any, response: any) {
       response.end("Hello World!");
       // terminate the process to stop CI pipeline from running forever
       server.close();
+      server.unref();
     }, 2000);
   });
 }

